sp800-63b/sec6_lifecycle.md

```diff
@@ -82,3 +82,32 @@ The CSP SHALL require subscribers to surrender or prove destruction of any physi
 
 Further requirements on the termination of PIV credentials are found in [[FIPS 201]](#FIPS201).
 
+### 6.5. Derived Authenticators
```

Comment:
total separate, but i could envision some linking of the 2 sections.  Remember, derived is the PIV use case.  I can only have the 2nd if I have the first.  And in some cases, you can lose the first and still get the second because of the proofing and biometric requirements.  Happy to discuss how to link, but prefer them to be separate for the PIV folks.